store/generated_key/eigenda/eigenda.go

```diff
@@ -20,6 +25,10 @@ type StoreConfig struct {
 
 	// total duration time that client waits for blob to confirm
 	StatusQueryTimeout time.Duration
+
+	// number of times to retry blob dispersal before returning a 503 to the client
```

Comment:
the returned error is not necessary 503. For example, when the last retry return a 500, due to chain re-org?

Reply:
Ohh good catch. Renamed to `PutRetries`: [956372a](https://github.com/Layr-Labs/eigenda-proxy/pull/193/commits/956372a6b949cd11109c65e9a8e81e729da7b360)